stable/phabricator/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: phabricator
-version: 0.4.5
+version: 0.4.6
```

Comment:
Since this bumps a minor version of a dependencies, we should think about bumping minor here as well.